src/components/TipInput.vue

```diff
@@ -150,6 +137,17 @@ export default {
     iconTip() {
       return this.isTipped ? iconTipped : iconTip;
     },
+    amount() {
+      return this.isRetip
+        ? this.tip.total_amount
+        : this.tip.retip_amount_ae;
+    },
+    title() {
+      if (this.isRetip) {
+        return this.isTipped ? 'Total tips (you tipped too)' : 'Total tips (click to retip the same URL)';
```

Comment:
Though this is wokring, IMHO all text data should be better in the template (as it could be easily translated, e.g. with filters)

Reply:
Does `vue-i18n` support translation based on filters? I don't see a significant difference to have texts here or in markup, it is more important to don't have to lot of business logic in components by extracting it to vuex store (at least this will make unit testing easier). To translate something in a template we are usually using `{{ t('translation-key') }}`, to do the same in the js section of a component we can use `this.$t('translation-key')`, so it shouldn't be more difficult to translate something in js section.